akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -200,8 +204,8 @@ abstract class UntypedPersistentActor extends UntypedActor with Eventsourced wit
    * @param events events to be persisted
    * @param handler handler for each persisted `events`
    */
-  final def persistAsync[A](events: JIterable[A])(handler: A ⇒ Unit): Unit =
```

Comment:
note that this signature was broken, and therefore I don't care about deprecating it